drafts/25-xxx-specifications.txt

```diff
@@ -28,53 +28,52 @@ discussions, these have included line continuation processing, comment
 handling, and tokenization for the preprocessor.)
 
 
-2. Formal specifications
-========================
+2 Lexical specifications
+------------------------
 
-2.1 Lexical specifications
---------------------------
-
-2.1.1 Lines
------------
+2.1 Lines
+---------
 
 li00. The Fortran preprocessor recognizes three distinct types of lines:
       preprocessing directives (and continuation lines thereof),
       Fortran comment lines, and Fortran source fragments.
 
-li10. A line that has a '#'
+li11. A line that has a '#'
       character as the first non-blank character of the line is a
       directive line (as required by C 2023 6.10.1 paragraph 2),
```

Comment:
We are using `C 2023` throughout to refer to the C spec, but the listing in the references is:

```
ISO/IEC 9899:2023 Programming languages -- C
                 (working draft N3096)
```

I see two options:
1. Replace all citations with `ISO/IEC 9899:2023`, or
2. Change the references to introduce a "short form", perhaps something like:

```
ISO/IEC 9899:2023 Programming languages -- C ("C 2023")
                 (working draft N3096)
```

Reply:
I'll update this PR.

Reply:
OK, I think I have updated the PR, but I am easily confused.